smart-contracts/osmosis/contracts/cl-vault/src/helpers/msgs.rs

```diff
@@ -114,15 +100,15 @@ fn osmosis_swap_exact_amount_in_msg(
 }
 
 fn cw_dex_execute_swap_operations_msg(
-    dex_router_address: Addr,
```

Comment:
This should be reverted so we keep type safety for callers of this function

Reply:
done